source/engine/physics/PhysicsConstants.h

```diff
@@ -16,19 +16,21 @@ constexpr bool EnableJointWarmstarting  = true;  // Start joint resolution from
 constexpr float WarmstartFactor = 0.75f;         // Percentage of previous impulse used to initialize constraints
 
 /** Contact Behavior */
-constexpr size_t EpaMaxIterations = 32u;                     // Number of iterations aftwerwhich to give up trying to find contact points
-constexpr float EpaTolerance = 0.00001f;                     // Error threshold for finding contact points
-constexpr float ContactBreakDistance = -0.01f;               // How aggressively contact points are removed/retained
-constexpr bool PreferSingleTangentContactBreak = false;      // Try to only discard only one contact due to tangential separation (potentially more stable for sliding scenarios)
-constexpr float MandatoryTangentContactBreakDistance = 0.1f; // Always discard contacts due to tangential separation beyond this threshold, regardless of PreferSingleTangentContactBreak
-constexpr float PenetrationSlop = 0.005f;                    // How far objects are allowed to penetrate
-constexpr bool EnableBaumgarteStabilization = true;          // Enable velocity-based position correction
-constexpr float BaumgarteFactor = 0.3f;                      // Bias factor for baumgarte stabilization [0, 1]
-constexpr bool EnableDirectPositionCorrection = false;       // Enable translation-based position correction
-constexpr float PositionCorrectionFactor = 0.2f;             // Percentage of mtv to use for direct position correction
-constexpr float MaxLinearPositionCorrection = 0.2f;          // Maximum translation distance for direct position correction
-constexpr bool EnableRestitutionSlop = true;                 // Enable zeroing of restitution below a threshold
-constexpr float RestitutionSlop = 0.001f;                    // Velocity threshold for applying restitution
+constexpr size_t EpaMaxIterations = 32u;             // Number of iterations aftwerwhich to give up trying to find contact points
+constexpr float EpaTolerance = 0.00001f;             // Error threshold for finding contact points
```

Comment:
Will you add [0,1] (or whatever the range is) to all of these like some of the others?